### PR TITLE
User repos settings updated

### PR DESCRIPTION
**Pull request by alishersadikov**
Remove unnecessary files.

### CODE REVIEW

**Comment by alishersadikov** [comment]
@davydovanton sorry I had committed unnecessary files before. Do we need anything else done for this issue?

File: Gemfile.lock [hunk 1]
```diff
@@ -1,428 +0,0 @@
-GIT
```
Comment (davydovanton) [inline Gemfile.lock]: I think we need to revert changes for `Gemfile.lock` file 😊

**Review by davydovanton — CHANGES_REQUESTED** [review]
Please, revert `Gemfile.lock` changes and fix merge conflicts. Thanks for your work 👍

**Comment by alishersadikov** [comment]
Sorry, my bad - still learning :)

**Pull request closed without merging**